spring-integration-core/src/main/java/org/springframework/integration/dsl/BaseIntegrationFlowDefinition.java

```diff
@@ -2829,6 +2840,26 @@ public B trigger(MessageTriggerAction triggerAction,
 		return handle(new ServiceActivatingHandler(triggerAction, "trigger"), endpointConfigurer);
 	}
 
+	/**
+	 * Adds one or more {@link ChannelInterceptor} implementations
```

Comment:
Method JavaDocs must be imperative style: https://github.com/spring-projects/spring-framework/wiki/Code-Style#javadoc-formatting